.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,155 @@
+                        the bug.\n
+                        If the issue can be replicated without code, please provide a clear, step-by-step description of
+                        the actions or conditions necessary to reproduce it. Any screenshots are also appreciated.\n
+                        Avoid including business logic or unrelated details, as this makes diagnosis more difficult.\n\n
+                        Whether it's a sequence of actions, code samples, or specific conditions, ensure that the steps
+                        are clear enough to be easily followed and replicated."
+      validations:
+          required: true
+
+    - type: "textarea"
+      id: "expected"
+      attributes:
+          label: "Expected Behavior"
+          description: "What did you expect to happen?"
+      validations:
+          required: true
+
+    - type: "textarea"
+      id: "current"
+      attributes:
+          label: "Current Behavior"
+          description: "What actually happened?\n\n
+                        Please include full errors, uncaught exceptions, stack traces, and relevant logs.\n
+                        If service/functions responses are relevant, please include wire logs."
+      validations:
+          required: true
+
+    - type: "textarea"
+      id: "workaround"
+      attributes:
+          label: "Possible Workaround"
+          description: "If you find any workaround for this problem - please, provide it here."
+      validations:
+          required: false
+
+    - type: "textarea"
+      id: "context"
+      attributes:
+          label: "Additional Information/Context"
+          description: "Anything else that might be relevant for troubleshooting this bug.\n
+                        Providing context helps us come up with a solution that is most useful in the real-world use case."
+      validations:
+          required: false
+
+    - type: "input"
+      id: "discussion_link"
+      attributes:
+          label: "Link to Discord discussion"
+          description: "Provide a link to the first message of bug's discussion on Discord.\n
+                        This will help to keep history of why this bug exists."
+      validations:
+          required: false
+
+    - type: "input"
+      id: "version"
+      attributes:
+          label: "Khoj version"
+          description: "If self-hosting - please make sure to run the latest version of Khoj before reporting any issues as 
+                        it may have already been fixed."
+      validations:
+          required: true
+
+    - type: "textarea"
+      id: "environment"
+      attributes:
+          label: "If self-hosting - environment details (OS name and version, Docker version, etc.)"
+          description: "Text you place here will be automatically wrapped into a code block."
+          render: "bash"
+      validations:
+          required: false
```

Comment:
This field seems a little redundant. Can be merged into the addition information field